src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py

```diff
@@ -57,6 +57,9 @@ def algorithm(self, parameters):
                 if response[0]['status'] == "ok" and len(response[0]['ids']) == len(jobIDs):
                     for docID in jobIDs:
                         self.fwjrAPI.updateArchiveUploadedStatus(docID)
+                logging.info("...successfully uploaded %d docs", len(jobIDs))
```

Comment:
These 3 lines need to go inside the `if`. Moreover, we could add an `else` to print the response returned by the server.
Typo on 'Arhcive'